docs-website/_docs/custom_checks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Custom Checks
+permalink: /docs/home/
```

Comment:
The page is not reachable on the actual website, I guess that's due to the fact that the permalink is wrong here.

Reply:
its not been released yet - when we cut a new release it'll update the website. Are you planning to be an early adopter of custom checks @particleflux ?

Reply:
Not actively planning it, was just wondering about that feature and tried to read up on the doc.
Noticed that it's broken at that point:

![2020-10-21_134408](https://user-images.githubusercontent.com/3686454/96715274-db47c980-1392-11eb-9fde-47a683fe8c62.png)